### ISSUE TITLE
Carthage installation and MPTweak

### ISSUE THREAD

**Issue by z4r** [mixpanel/mixpanel-iphone#451]
Hi guys,
probably it's my fault, but I really cant understand how to import any symbol related to `MPTweakInline.h`.
I have this issue with the v2.9.3 and v2.9.4 (and master) installed via Carthage.

The project I'm working on is written in Swift. 
If I install Mixpanel via Cocoapods I'm able to bridge with something like this because I've visibility of `Mixpanel/MPTweakInline.h`:

```objective-c
#import "MPTweaksWrapper.h"
#import <Mixpanel/MPTweakInline.h>

@implementation MPTweaksWrapper
+ (NSString *)subscriptionID {
    return MPTweakValue(@"subscriptionID", @"MY_ID");
}
@end
```

and call it from Swift 

```swift
let subscriptionID = MPTweaksWrapper.subscriptionID()
```
 
With the Carthage installation I've visibility only for the main header `Mixpanel.h`.

![Mixpanel Header](https://cloud.githubusercontent.com/assets/639459/13325925/336e97f6-dbe4-11e5-9bd1-7426f7b8cad5.png)

Pls help me, because I don't want to add Cocoapods just for tweaks.

**Comment by z4r** [comment]
My only way to have it working on my fork is move every related with `MPTweakInline.h` in the public section of Headers in Build Phases

![schermata 2016-02-27 alle 00 11 11](https://cloud.githubusercontent.com/assets/639459/13367972/fb068cb2-dce6-11e5-9517-2b026c861076.png)

And then in the declared Umbrella Header `Mixpanel.h` I added the line:

```objective-c
#import "MPTweakInline.h" 
```

Of course in my objc `MPTweaksWrapper` I can simply import just the Umbrella

```objective-c
#import "MPTweaksWrapper.h"
#import <Mixpanel/Mixpanel.h>

@implementation MPTweaksWrapper
+ (NSString *)subscriptionID {
    return MPTweakValue(@"subscriptionID", @"MY_ID");
}
@end
```

Am I right with this solution? If you want to try something I committed everything on the master branch of my fork.

**Comment by samgreen** [comment]
Hey @z4r you are doing everything correctly. This was an oversight on our part with the Carthage release. We'll get this patched up and released ASAP

**Comment by z4r** [comment]
Hi @samgreen,
thx for reply.

Just to plan our release timeline, the order of your ASAP is something closer to a week or to a month?

**Comment by samgreen** [comment]
Hi @z4r unfortunately we don't typically comment publicly on release timelines, in case something happens that prevents it. In the short term your fork should be fine to release with. This is a top priority for the SDK and I can guarantee it will be in the next release, however I can't give you a concrete timeline. I will say that I highly doubt that it would be a month before the next release.

Is that helpful?

**Comment by z4r** [comment]
Very helpful!
My only concern was my fork too far from your main repo, but I can obviously wait till next release.

Thank you.

**Comment by JagCesar** [comment]
I get `Umbrella header 'Mixpanel.h' not found` when I'm trying to compile an watchOS app that has Mixpanel compiled with Carthage added to it. Can it be related to this issue?

![screen shot 2016-03-01 at 21 12 26](https://cloud.githubusercontent.com/assets/923548/13440448/5ac891dc-dff2-11e5-8480-d259be9c3612.png)

**Comment by kballard** [comment]
It's getting close to a month now. Any chance this will be fixed soon?

**Comment by PabloLerma** [comment]
@samgreen would a PR be helpful?

**Comment by z4r** [comment]
Hi @PabloLerma,
if they need a PR I've it ready on my fork.

**Comment by samgreen** [comment]
@PabloLerma Would love a PR!

**Comment by samgreen** [comment]
@JagCesar Can you open a new issue? I think your issue is unrelated.

**Comment by samgreen** [comment]
This will be resolved with #463

**Comment by samgreen** [comment]
This is resolved in v2.9.5 which is now released.
